### ISSUE TITLE
Module.json has error on generation

### ISSUE THREAD

**Issue by nWidart** [pingpong-labs/modules#130]
Hello,

After generating a module, `module.json` has this:

``` json
"providers": [
        "$MODULE_NAMESPACE$\\Recipe\\Providers\\RecipeServiceProvider"
    ],
```

**Comment by nWidart** [comment]
Any chance you could look at this and #129 as this breaks the generation in Asgard too. :smile: 

Thanks!

**Comment by gravitano** [comment]
Have you updated your config file from this package ? There is a new config `namespace`.

**Comment by nWidart** [comment]
I was indeed missing that config value.
I've added `'namespace' => 'Modules',` at the first level of the array, though I still get the same issue.

Also, since in your config file I see `App\Modules` as default namespace, it should at least have used that one, which it didn't.

**Comment by gravitano** [comment]
Hmm weird. I just tried generating module and all works fine. Try update the package.

**Comment by nWidart** [comment]
Just ran composer update, but I was already using the latest version.

Tried a new `module:make` and I still get that $module_namespace$.

I'm using: `"pingpong/modules": "2.0.*@dev",`

**Comment by gravitano** [comment]
I've just tried on a fresh laravel and all goes well. I've tried changing the config namespace and module path and all also goes well. Maybe There are something miss in your app.

**Comment by nWidart** [comment]
Well I have a custom command that wraps around your `module:make`, but for this test, I've used `module:make` directly, so whatever I have shouldn't intervene.

I don't see what could go wrong here. When I look at `vendor/pingpong/modules/src/config.php` I do see the `namespace` key with `App\Modules`, so it should at the very least use that.

Don't know why it isn't.
Again, since I'm directly using `module:make`, I'm calling your command where I didn't change anything.

I also just did a clean install, and indeed it works there.
The only difference I see is the commit hash, on a clean laravel app I have `8134aebfdfbec9382cd48304e8d2f06657f54291`, and on the apps where I have the problem `b6f4b45d51f6d99a1f14b1c4b43759517b9560fc`, although I ran composer update.

Update:

Had to add ` 'MODULE_NAMESPACE'` in the `composer` key of config file.

``` php
'composer'      => [
                'LOWER_NAME', 'STUDLY_NAME', 'VENDOR', 'AUTHOR_NAME', 'AUTHOR_EMAIL',
                'MODULE_NAMESPACE'
            ],
```

Same for the `json` key.

Keeping a changelog would remove all these issues of keeping of to date to your latest changes, since you forgot about those changes too.
